pytest/tests/mocknet/helpers/load_test_utils.py

```diff
@@ -0,0 +1,233 @@
+import random
+import sys
+import time
+
+import base58
+from rc import pmap
+
+# Don't use the pathlib magic because this file runs on a remote machine.
```

Comment:
Are you planning on addressing this in this PR? If not what is the plan?

Reply:
Directory structure on remote machines is different. We are not copying the `pytests` directory (hm, why not? :thinking: ), but only the `pytests/tests/mocknet/helpers/` and `pytests/lib` parts.